docs/source/project_documentation/home_page_index.rst

```diff
@@ -0,0 +1,158 @@
+==========================
+Architecture-as-Code (AaC)
+==========================
+
+Get started now by reading the `User Guide Documentation <./project_documentation/user_guide/index.html>`_.

```

Comment:
```suggestion
Get started now by reading the `AaC User Guide Documentation <./project_documentation/user_guide/index.html>`_.

```

The link referencing at the end of this would not work. You would need to update it to the actual website documentation link.